zach_kaz/forloopbasic2.py

```diff
@@ -0,0 +1,70 @@
+def biggie(arr):
+    for i in arr:
+        if i>0:
+            arr[i]="big"
```

Comment:
here i is the actual value and not the index. So if the input contains value greater than the length of the array then this line will crash your code